features/ifttt/src/main/java/org/opennms/features/ifttt/config/TriggerPackage.java

```diff
@@ -64,7 +66,7 @@ public void setCategoryFilter(final String categoryFilter) {
 
     @XmlAttribute
     public String getUeiFilter() {
-        return ueiFilter != null ? ueiFilter : ".*";
+        return Strings.isNullOrEmpty(ueiFilter) ? ".*" : ueiFilter;
```

Comment:
I was talking about using the empty string instead of the match-all regex... Not sure if this works out.